subprojects/kotlin-dsl/src/test/kotlin/org/gradle/kotlin/dsl/execution/LexerTest.kt

```diff
@@ -71,6 +71,57 @@ class LexerTest {
         )
     }
 
+    @Test
+    fun `ignores label definitions`() {
+
+        assertThat(
+            lex(
+                "kuku@ someblock {}",
+                buildscript, plugins
+            ),
+            equalTo(
+                Packaged(
+                    null,
+                    LexedScript(
+                        listOf(),
+                        listOf(),
+                        listOf()
+                    )
+                )
+            )
+        )
+    }
+
+    @Test
+    fun `skips annotation parsing inside blocks`() {
+
+        assertThat(
+            lex(
+                "kuku@ plugins.withType<GroovyBasePlugin> {\n" +
```

Comment:
💅 I would remove the label and reduce the tested snipped to more clearly convey what is tested

Reply:
Fixed